docs/source/load_hub.rst

```diff
@@ -0,0 +1,85 @@
+
+.. seealso::
+
+   Take a look at :class:`datasets.DatasetInfo` for a full list of attributes you can use with ``dataset_builder``.
+
+Once you are happy with the dataset you want, load it in a single line with :func:`datasets.load_dataset`:
+
+.. code-block::
+
+   >>> from datasets import load_dataset
+   >>> dataset = load_dataset('glue', 'mrpc', split='train')
+
+Select a split
+--------------
+
+A split is a specific subset of the dataset like ``train`` and ``test``. Make sure you select a split when you load a dataset. If you don't supply a ``split`` argument, 🤗 Datasets will only return a dictionary containing the subsets of the dataset.
+
+.. code-block::
+
+   >>> from datasets import load_dataset
+   >>> datasets = load_dataset('glue', 'mrpc')
+   >>> print(datasets)
+   {train: Dataset({
+       features: ['idx', 'label', 'sentence1', 'sentence2'],
+       num_rows: 3668
+   })
+   validation: Dataset({
+       features: ['idx', 'label', 'sentence1', 'sentence2'],
+       num_rows: 408
+   })
+   test: Dataset({
+       features: ['idx', 'label', 'sentence1', 'sentence2'],
+       num_rows: 1725
+   })
+   }
+
+Select a configuration
+----------------------
+
+Some datasets, like the `General Language Understanding Evaluation (GLUE) <https://huggingface.co/datasets/glue>`_ benchmark, are actually made up of several datasets. These sub-datasets are called **configurations**, and you must explicitly select one when you load the dataset. If you don't provide a configuration name, 🤗 Datasets will raise a ``ValueError`` and remind you to select a configuration.
+
+❌ Incorrect way to load a configuration:
+
+.. code-block::
+
+   >>> from datasets import load_dataset
+   >>> dataset = load_dataset('glue')
+   ValueError: Config name is missing.
+   Please pick one among the available configs: ['cola', 'sst2', 'mrpc', 'qqp', 'stsb', 'mnli', 'mnli_mismatched', 'mnli_matched', 'qnli', 'rte', 'wnli', 'ax']
+   Example of usage:
+           `load_dataset('glue', 'cola')`
+
+✅ Correct way to load a configuration:
+
+.. code-block::
+
+   >>> dataset = load_dataset('glue', 'sst2')
+   Downloading and preparing dataset glue/sst2 (download: 7.09 MiB, generated: 4.81 MiB, total: 11.90 MiB) to /Users/thomwolf/.cache/huggingface/datasets/glue/sst2/1.0.0...
+   Downloading: 100%|██████████████████████████████████████████████████████████████| 7.44M/7.44M [00:01<00:00, 7.03MB/s]
+   Dataset glue downloaded and prepared to /Users/huggignface/.cache/huggingface/datasets/glue/sst2/1.0.0. Subsequent calls will reuse this data.
+   >>> print(dataset)
+   {'train': Dataset(schema: {'sentence': 'string', 'label': 'int64', 'idx': 'int32'}, num_rows: 67349),
+       'validation': Dataset(schema: {'sentence': 'string', 'label': 'int64', 'idx': 'int32'}, num_rows: 872),
+       'test': Dataset(schema: {'sentence': 'string', 'label': 'int64', 'idx': 'int32'}, num_rows: 1821)
+   }
```

Comment:
What do you think about showing the end-user how the can get the whole list of configurations, e.g. something like:

```python
from datasets import get_dataset_config_names

configs = get_dataset_config_names("glue")
print(configs)
# ['cola', 'sst2', 'mrpc', 'qqp', 'stsb', 'mnli', 'mnli_mismatched', 'mnli_matched', 'qnli', 'rte', 'wnli', 'ax']
```